Code/bin/main.py

```diff
@@ -23,31 +23,41 @@ def loss(original:Sequence[Label],applied:Sequence[Label], miss_w=1.0,extra_w=1.
     return miss_w*len(misses) + extra_w*len(extras)
 
 
-def run(plays:Sequence[tuple[DataFrame,str]]):
+def label_plays(plays:Sequence[tuple[DataFrame,str]]):
+    ## TODO FIXME: use actual labeling function according to vectors.
```

Comment:
@iristaub
I made the wrapper for using the function that labels a fragment
we need to replace `convert_labels(parse_fragment(row.Fragment))` with the new functi0on you are going to write